packages/expo-modules-core/package.json

```diff
@@ -41,6 +41,10 @@
   "dependencies": {
     "invariant": "^2.2.4"
   },
+  "peerDependencies": {
+    "react": "*",
+    "react-native": "*"
+  },
```

Comment:
wonder if we need
```
  "peerDependenciesMeta": {
    "react": {
      "optional": true
    },
    "react-native": {
      "optional": true
    }
  }
```

to prevent npm install a nested react/react-native inside. not pretty sure but maybe we can just merge and see if it happens

Reply:
At least for `pnpm` and `yarn` classic it doesn't seem to happen (and I'd be very surprised if it did off of a `*` range). If it happens, I'd say, we can counteract this, but generally, peer dependencies on `react` and `react-native` are extremely common, so if this was to affect us, it'd also affect others

I'll keep an eye on this during the beta via Discord feedback though 👍